source/hp/refinement.cc

```diff
@@ -634,7 +634,9 @@ namespace hp
             if (cell->future_fe_index_set())
               {
                 predicted_errors[cell->active_cell_index()] *=
-                  std::pow(gamma_p, future_fe_degree - cell->get_fe().degree);
+                  std::pow(gamma_p,
+                           static_cast<int>(future_fe_degree -
+                                            cell->get_fe().degree));
```

Comment:
```suggestion
                           int(future_fe_degree) -
                           int(cell->get_fe().degree));
```
casting to an `int` after the fact relies on some implementation-defined behavior - casting before subtracting is unambiguous.